locking/management/commands/clean_expired_locks.py

```diff
@@ -1,22 +1,22 @@
-from optparse import make_option
-
-from django.core.management.base import NoArgsCommand
+from django.core.management.base import BaseCommand
 
 from locking.models import NonBlockingLock
 
 from logging import getLogger
 logger = getLogger(__name__)
 
 
-class Command(NoArgsCommand):
+class Command(BaseCommand):
```

Comment:
Changed this because it was deprecated in 1.8 and gave failing tests. More info https://docs.djangoproject.com/en/1.9/howto/custom-management-commands/ below when looking for NoArgsCommand.